purchase_operating_unit/models/procurement.py

```diff
@@ -10,18 +10,20 @@
 class ProcurementOrder(models.Model):
     _inherit = 'procurement.order'
 
-    @api.one
+    @api.multi
     @api.constrains('purchase_line_id')
```

Comment:
Question: I don't use v10 anymore, but doesn't the constraint now also apply to location_id?  I doubt it would trigger in practice anyway.  On second thoughts, that gets problematic when considering other modules that might raise procurement.